src/SnapKeyring.ts

```diff
@@ -414,14 +415,15 @@ export class SnapKeyring extends EventEmitter {
     // where the Snap responds before we have a chance to create it.
     const promise = this.#createRequestPromise<Response>(requestId, snapId);
 
-    const response = await this.#submitSnapRequest({
+    const request = {
       snapId,
       requestId,
       account,
       method: method as EthMethod,
       params,
       chainId,
-    });
+    };
+    const response = await this.#submitSnapRequest(request);
```

Comment:
Was your intention here also to add a log?

Reply:
Damn! That's a left over from my previous implementation 🙈

That was my initial plan yes, but I feel like it's better to have it in `submitSnapRequest` directly actually.

I'll revert that, thanks!